packages/@guidesmiths/cuckoojs-cli/src/lib/runners/bash.runner.husky.ts

```diff
@@ -3,18 +3,25 @@ import {GenericRunner} from './generic.runner';
 import {join} from 'path';
 
 export class BashRunnerHusky extends GenericRunner {
-	private static buildCommitLintPath(name: string): string {
-		return join(process.cwd(), name, '.husky/commit-msg');
+	private static getHuskyHookFile(name: string, hook: string): string {
+		return join(process.cwd(), name, '.husky', hook);
 	}
 
 	constructor() {
 		super('npx');
 	}
 
-	public async runHuskyCommit(folderName: string) {
-		const fullPath = BashRunnerHusky.buildCommitLintPath(folderName);
+	public async addHuskyPreCommit(folderName: string) {
+		const fullPath = BashRunnerHusky.getHuskyHookFile(folderName, 'pre-commit');
```

Comment:
Should this be `pre-commit` or `commit-msg`?

Reply:
done

Reply:
Thanks!